lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3544,6 +3544,14 @@ namespace {
       auto def = dyn_cast<clang::ClassTemplateSpecializationDecl>(
           decl->getDefinition());
       assert(def && "Class template instantiation didn't have definition");
+
+      // If this type is fully specialized (i.e. "Foo<>" or "Foo<int, int>"),
+      // bail to prevent a crash.
+      // TODO: we should be able to support fully specialized class templates.
+      // See SR-13775 for more info.
+      if (def->getTypeAsWritten())
```

Comment:
I tried adding some logic here to allow us to continue importing types like `WrapperWithSpecialization` but that required lots of special cases and messy conditions that could allow patterns that still crash. I think it's best to just disable this entirely until I can properly fix it.